Dockerfile

```diff
@@ -29,4 +29,6 @@ COPY --from=builder /opt/grobid-source/grobid-trainer/resources /opt/grobid-sour
 COPY scripts /opt/scripts
 ENV PATH /opt/scripts:$PATH
 
-ENV JAVA_OPTS -Xmx1G
+ENV JAVA_OPTS=-Xmx1G
+
+LABEL org.elifesciences.dependencies.grobid="${grobid_tag}"
```

Comment:
also `org.opencontainers.image.revision` given the commit is composed into the tag but not available in a clean form?